doc/USAGE.md

```diff
@@ -0,0 +1,187 @@
+- Update the `managers_xpubs`: it will contain all the manager xpubs created in the ceremony
+- Update the `cosigner_keys`: it will contain all the cosigner keys created in step 3
+- Update the `stakeholder_config`: update `xpub` to match your xpub; this section must be there only if you're a stakeholder, or a manager-stakeholder. Leave the watchtowers section as it is, it's not used (yet!)
+- Update the `manager_config`: update `xpub` to match your xpub; the `cosigner` key to match your cosigner key otained in step 3; this section must be there only if you're a manager, or a manager-stakeholder
+- Update the `bitcoind_config` `addr` to the address of your bitcoin RPC. In regtest defaults to `127.0.0.1:18832`. The one contained in the regtest config is the correct one if you're using the regtest manager script, as outlined below
+
+### 5. Spinning up revaultd
+Time to start revaultd! This will create the database and the watchonly wallet. After you run `revaultd` for the first time, the database and the watchonly wallet must be deleted if you wish to change one or more of the managers/stakeholders/cosigners keys.
+To make things easier, we'll create a couple of aliases:
+```
+alias rd="cargo run --release --bin revaultd -- --conf our_modified_config.toml"
+alias rcli="cargo run --release --bin revault-cli -- --conf our_modified_config.toml"
+```
+Now starting `revaultd`:
+```
+rd
+```
+Revaultd will print the noise key used when starting. Collect all the managers' noise keys, put them in the cosigner configuration (the one we created in step 2).
+
+If no error was printed, congrats! You succesfully started `revaultd`.
+
+Now kill it using `CTRL+C`, open the `revaultd` configuration again and set `daemon=true`: we don't want to occupy and entire terminal for `revaultd`!
+
+Start `revaultd` once again:
+```
+rd
+```
+
+Go back to the parent directory for the next step
+```
+cd ..
+```
+
+### 6. Starting the cosigners
+
+Once you have updated the manager noise keys in the cosigner config, it's time to start the cosigner with the right config:
+```
+cd cosignerd
+cargo run --release -- --conf our_modified_config.toml
+```
+
+Go back to the parent directory for the next step
+```
+cd ..
+```
+
+### 7. Sending commands using the cli
+Let's go back to the revaultd directory and ask for info:
+```
+cd revaultd
+rcli getinfo
+```
+
+You should see something like:
+```
+{"id":"revault-cli-7957","jsonrpc":"2.0","result":{"blockheight":172,"network":"regtest","sync":1.0,"vaults":1,"version":"0.0.2"}}
+```
+
+### 7. Wait, CLI only?!
+Nah. We also have a GUI. Check out [revault-gui](https://github.com/revault/revault-gui) :)
+
+## Testing
+For testing you may find useful the [regtest manager script](../contrib/regtest_manager.sh). You will need to tweak the configuration we created in step 4 a bit, as the RPC address of the bitcoin node spawn by the script is `127.0.0.1:9001`.
```

Comment:
I wanted to get rid of this script in the repo as it seems to only be useful to me and confusing to others, but finally maybe not?

Reply:
Not sure if it's actually useful to people (I've never tried it <3), but it's not hurting anyone, so I would just keep it there